doc/modules/classes.rst

```diff
@@ -1183,6 +1183,7 @@ See the :ref:`metrics` section of the user guide for further details.
    preprocessing.Binarizer
    preprocessing.FunctionTransformer
    preprocessing.Imputer
+   preprocessing.KBinsDiscretizer
```

Comment:
This is the rendered page:

<img width="845" alt="screen shot 2017-02-26 at 3 43 09 am" src="https://cloud.githubusercontent.com/assets/1865885/23339386/f75835ca-fbd5-11e6-99f7-78de1796d8d7.png">